python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2187,6 +2187,14 @@ def from_tensorflow(self, graph, layout="NHWC", shape=None, outputs=None):
         missing_operators = self._parse_import_prerequisites(graph)
 
         if missing_operators:
+            # TODO: ReadVariableOp gets removed when graph is frozen.
+            # Add list of other operators as well which get removed
+            # and are not needed for inference.
+            # Other approach is instead of raising error, we can freeze the graph.
+            if 'ReadVariableOp' in missing_operators:
```

Comment:
Is there any other op falling into this case? If so, I suggest we create a list containing all these ops.

Reply:
Yes, there is.
I will compile the list and add it in.

Reply:
Added the other operators as well